docs/develop-logseq.md

```diff
@@ -24,6 +24,21 @@ yarn watch
 
 Then open the browser <http://localhost:3001>.
 
+### REPL setup
+
+#### VSCode + Calva
+With ```yarn watch``` running, it should prints ``shadow-cljs - nREPL server started on port 8701``
+
+You may connect to the nREPL server with:
+
+``cmd + shift + p`` -> ``Calva: Connect to a running nREPL server`` -> ``logseq`` -> ``shadow-cljs``->``:app`` ->``localhost:8701``
```

Comment:
**non-blocking**: Or open Calva by clicking "REPL ⚡️" from the status bar.
`cmd` key is only for macOS users.